compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Type.hs

```diff
@@ -32,7 +32,7 @@ freeVars e = go Set.empty e Set.empty
         TBuiltin _ -> acc
         TForall (v, _k) s -> go (Set.insert v boundVars) s acc
         TStruct fs -> foldl' (\acc (_, t) -> go boundVars t acc) acc fs
-        TNat _ -> Set.empty
+        TNat _ -> acc
```

Comment:
The argument of `TNat` can contain type variables as well, can't it?

Reply:
It contains only a `TypeLevelNat`, which is a newtype over `Int`, so no.

Reply:
Oh right, I confused it with `TNumeric`. My bad.